test/e2e/e2e.go

```diff
@@ -1298,13 +1298,9 @@ spec:
 		err = wait.PollImmediate(retryInterval, retryTimeout, targetExternalContainerAndTest(targetNode, pod1Name, podNamespace.Name, true, []string{egressIP.String()}))
 		framework.ExpectNoError(err, "Step: 5. Check connectivity to the allowed IP and verify it has the egress IP, failed, err: %v", err)
 
-		// TODO: in the future once we only have shared gateway mode: implement egress firewall so that
-		// pods that have a "deny all 0.0.0.0/0" rule, still can connect to the Kubernetes API service
-		// and re-enable this check
-
-		// ginkgo.By("6. Check connectivity to the kubernetes API IP and verify that it works")
-		// err = wait.PollImmediate(retryInterval, retryTimeout, targetAPIServiceAndTest(podNamespace.Name, []string{pod1Name, pod2Name}))
-		// framework.ExpectNoError(err, "Step 6. Check connectivity to the kubernetes API IP and verify that it works, failed, err %v", err)
+		ginkgo.By("6. Check connectivity to the kubernetes API IP and verify that it works")
```

Comment:
nice!

Reply:
Actually, I realize I need a check on the mode this is running in, because I couldn't fix it in local gateway mode.